src/panel/PanelManager.jsx

```diff
@@ -212,12 +216,39 @@ export default class PanelManager extends React.Component {
   }
 
   render() {
-    const { ActivePanel, options, panelSize } = this.state;
-
-    return <PanelContext.Provider value={{ size: panelSize, setSize: this.setPanelSize }} >
-      <div className="panel_container">
-        <ActivePanel {...options} />
-      </div>
-    </PanelContext.Provider>;
+    const { ActivePanel, options, panelSize, isPanelVisible } = this.state;
+    const { searchBarInputNode, searchBarOutputNode } = this.props;
+
+    return <div>
+      <Suggest
+        inputNode={searchBarInputNode}
+        outputNode={searchBarOutputNode}
+        withCategories
+        onChange={query => {
+          this.setState({
+            isPanelVisible: ActivePanel === ServicePanel && query.length === 0,
+          });
+        }}
```

Comment:
As is, the `<Suggest>` component will be re-rendered for each input: its `currentQuery` will not be persisted and the successive queries aren't properly aborted.
These weird race conditions may also explain why the CI is failing.

Maybe we should check the previous state in `setState`, to check if the state really needs to be changed ?
Or maybe `React.memo` should be used in Suggest.jsx ?